lib/dsc-lib/src/configure/config_doc.rs

```diff
@@ -466,4 +511,69 @@ mod test {
 
         assert!(result.is_ok());
     }
+
+    #[test]
+    fn test_deserialize_resources_array() {
```

Comment:
It's ok to keep a few unit tests here, but would prefer to also add some Pester tests.

Reply:
Done.